test/local/isolated-pool-multi-users-utilisation-rates.test.ts

```diff
@@ -65,7 +65,7 @@ describe('Pool with fixed interests rates', () => {
       await time.increase(time.duration.years(1));
 
       // poolBalance = fromWei(await provider.getBalance(pool.address));
```

Comment:
Commented out code

Reply:
sorry, this test shouldn't be included in commit. Removing. This logic is tested in `pool-health.test.ts`